cmd/util.go

```diff
@@ -33,13 +33,6 @@ func init() {
 	statFunc = os.Stat
 }
 
-func printlnWarn(msg string) {
-	fmt.Fprintf(os.Stderr, "[%s] %s\n",
-		colors[check.WARN].Sprintf("%s", check.WARN),
-		msg,
-	)
-}
-
 func sprintlnWarn(msg string) string {
```

Comment:
I think we could delete `sprintlnWarn` this function too now?

Reply:
It's not relevant to this issue but yes, this function is not called by anyone so I'm deleting it.